app/models/keyword_user.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class KeywordUser < ApplicationRecord
+  belongs_to :keyword
+  belongs_to :user
```

Comment:
let's use the `inverse of` as [explained in compass](https://nimblehq.co/compass/development/code-conventions/ruby/ruby-on-rails/#convention-13)

```suggestion
  belongs_to :keyword, inverse_of: :keyword_users
  belongs_to :user, inverse_of: :keyword_users
```

Same for the other models~

Reply:
I removed this file and add `inverse_of` to `keywords.rb` 🙏